public/app/features/dashboard-scene/pages/DashboardScenePageStateManager.test.ts

```diff
@@ -1184,3 +1212,446 @@ const customHomeDashboardV2Spec = {
     },
   },
 };
+
+const v1ProvisionedDashboardResource = {
```

Comment:
This is huge 😓 This file is big enough, let's extract this into a __fixture__ file :D

Reply:
yes, i will but not today hahahahahah :D @Clarity-89 - cold you do it in your pr https://github.com/grafana/grafana/pull/103892? I guess we can slim down those fixtures a LOT given Repository has plenty of optionals.

Reply:
Yeah, we can do it in a follow-up, but not today either :)

Reply:
TY